Frameworks/include/CoreTextInternal.h

```diff
@@ -85,14 +86,19 @@ inline void _SafeRelease(T** p) {
 }
 @end
 
-@interface _CTFrame : NSObject {
-@public
+#pragma region CTFrame
+struct __CTFrame : CoreFoundation::CppBase<__CTFrame> {
+    __CTFrame() : _lines(CFArrayCreateMutable(kCFAllocatorDefault, 0, &kCFTypeArrayCallBacks)) {
```

Comment:
sadly due to the extensive usage of the internal vars outside the actual class, these had to go public (internally) :(